forge-gui/res/cardsfolder/upcoming/xanders_wake.txt

```diff
@@ -3,6 +3,6 @@ ManaCost:1 B
 Types:Enchantment
 T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Creature.YouCtrl | TriggerZones$ Battlefield | Execute$ TrigDraft | ActivationLimit$ 1 | TriggerDescription$ Whenever one or more creatures you control die, draft a card from CARDNAME's spellbook. This ability triggers only once each turn.
 SVar:TrigDraft:DB$ Draft | Spellbook$ Acquisitions Expert,Asylum Visitor,Blade Juggler,Bloodthirsty Aerialist,Boneclad Necromancer,Hoard Robber,Malakir Blood-Priest,Malakir Cullblade,Morbid Opportunist,Slaughter Specialist,Tavern Swindler,Thieves' Guild Enforcer,Tithebearer Giant,Vengeful Warchief,Yuan-Ti Fang-Blade
-DeckHints:Ability$Sacrifice
-DeckHas:Type$Vampire|Rogue
+DeckHas:Type$Human|Rogue|Vampire|Wizard|Tiefling|Cleric|Warrior|Giant|Orc|Snake|Zombie|Treasure & Ability$Token|Sacrifice|LifeGain|Counters
+DeckHints:Ability$Party|LifeGain|Discard & Type$Cleric|Rogue|Warrior|Wizard
```

Comment:
Shoot, sorry. We should have kept the Sacrifice
`DeckHints:Ability$Party|LifeGain|Discard|Sacrifice & Type$Cleric|Rogue|Warrior|Wizard`